src/Parser.js

```diff
@@ -179,7 +181,20 @@ module.exports = class Parser {
           out += top ? this.renderer.paragraph(body) : body;
           continue;
         }
+
         default: {
+          // Run any renderer extensions
+          tokenParsed = false;
+          if (this.options.extensions) {
+            Object.values(this.options.extensions).forEach(function(extension, index) {
```

Comment:
we should also create a map with `name` as the key to prevent looping here as well.

Reply:
Ok. Shouldn't be too hard.

Reply:
This is done now.